scripts/create-user.sh

```diff
@@ -89,6 +89,10 @@ check_params_and_environment() {
     abort_usage "Org must be defined"
   fi
 
+  if ! cf orgs >/dev/null 2>&1; then
+    abort "You need to be logged into CF CLI as admin user"
```

Comment:
This doesn't actually test if the user is admin or not. Can we change the message so it deals only with logged in or not?